src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Dto/AttributeForm.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright 2022 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Dto;
+
+class Form
+{
+    public $languages = [];
+
+    private function __construct(
+        array $languages
+    ) {
+        foreach ($languages as $language => $values) {
+            $this->languages[$language] = language::fromLanguage($values);
```

Comment:
1. As per my previous suggestion. The Language value object should itself also be aware about the language it represents. Its perfectly okay to key your lang-collection on the language as well. 
2. Typo `language::fromLanguage` Classname starts with lower case `l`

Reply:
I will add a language property to make aware about the language.